### DISCUSSION TITLE
Gitleaks detecting leaks after encrypting and pushing files to repo

### DISCUSSION THREAD

**Discussion by nitishpatnaik1903** [zricethezav/gitleaks#1272] [Q&A]
Even after encrypting api keys, passwords in project, and pushing code to main branch, Gitleaks is detecting the leaks. 
I have used git-crypt to encrypt the files.

Commands used ->
gitleaks detect --report-path gitleaks-report.csv -s [project-dir] -v 
git-crypt status -e
git-crypt lock
git-crypt unlock [path to keyfile]

Any suggestions to fix this ?

**Comment by rgmz** [comment]
It's impossible to say without more information. 

- What is being detected?
- Is the detection a false or true positive?
